src/xdocs/checks/blocks/emptycatchblock.xml

```diff
@@ -64,27 +64,37 @@
         </source>
         <p id="Example1-code">Such empty blocks would be both suppressed:</p>
         <source>
-try {
-  throw new RuntimeException();
-} catch (RuntimeException expected) {
-} // violation
+public class Example1 {
+    private void exampleMethod1() {
+        try {
+            throw new RuntimeException();
+        } catch (RuntimeException expected) {
+        } // violation
+    }
 
-try {
-  throw new RuntimeException();
-} catch (RuntimeException ignore) {
-  // no handling
-} // ok, catch block has comment
+    private void exampleMethod2() {
+        try {
+            throw new RuntimeException();
+        } catch (RuntimeException ignore) {
+            // no handling
+        } // ok, catch block has comment
+    }
 
-try {
-  throw new RuntimeException();
-} catch (RuntimeException o) {
-} // violation
+    private void exampleMethod3 () {
+        try {
+            throw new RuntimeException();
+        } catch (RuntimeException o) {
+        } // violation
+    }
 
-try {
-  throw new RuntimeException();
-} catch (RuntimeException ex) {
-  // This is expected
-} // ok
+    private void exampleMethod4 () {
+        try {
+            throw new RuntimeException();
+        } catch (RuntimeException ex) {
+            // This is expected
+        } // ok
```

Comment:
please remove all ` // ok` comments 
it is good to jave comeents like `// ok, catch block has comment` that has some explanation, but if no explanation then no comment is required.